src/controller/ReadInteraction.h

```diff
@@ -45,7 +45,10 @@ template <typename DecodableAttributeType>
 CHIP_ERROR ReadAttribute(Messaging::ExchangeManager * aExchangeMgr, const SessionHandle sessionHandle, EndpointId endpointId,
                          ClusterId clusterId, AttributeId attributeId,
                          typename TypedReadAttributeCallback<DecodableAttributeType>::OnSuccessCallbackType onSuccessCb,
-                         typename TypedReadAttributeCallback<DecodableAttributeType>::OnErrorCallbackType onErrorCb)
+                         typename TypedReadAttributeCallback<DecodableAttributeType>::OnErrorCallbackType onErrorCb,
```

Comment:
could we  make this a bit smaller than 10 parameters long? this hurts readability of something like:

```
err = ReadAttribute(...., 123, 345, false, ...)
```

since there is no clear indication what 123 or 345 or what false is.

Reply:
agree on this, we would change it later, just remove redundant parameter from the previous pr.